packages/ui-app/src/util/toShortAddress.ts

```diff
@@ -2,6 +2,8 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
+import isUndefined from '@polkadot/util/is/undefined';
+
 export default function toShortAddress (address: string): string {
-  return `${address.slice(0, 7)}…${address.slice(-7)}`;
+  return isUndefined(address) ? '' : `${address.slice(0, 7)}…${address.slice(-7)}`;
```

Comment:
Use toShortAddress as per the previous review comments.

Reply:
Yes I have, this is the `toShortAddress` function that I'm using now